### PR TITLE
Sources supporting tests from file, generate and parse a single JSON record instead of a whole file

### PR DESCRIPTION
**Pull request by jedrz**
* Sources handle a single record instead of whole raw test data with multiple records
  * `TestDataParser` replaced with `TestRecordParser` that turns a test JSON record into source record 
  * `TestDataGenerator#generateTestData` returns JSON test records
* Temporarily a JSON test record is serialized directly without source node id and timestamp (to be added in next MRs).
* Scenario testing logic from routes extracted to a new service.

### CODE REVIEW

File: engine/flink/schemed-kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/schemedkafka/source/flink/TestFromFileSpec.scala [hunk 1]
```diff
@@ -57,8 +58,9 @@ class TestFromFileSpec extends AnyFunSuite with Matchers with LazyLogging {
       .encoder(BestEffortJsonEncoder.defaultForTests.encode).apply(inputMeta)
       .mapObject(_.add("key", Null)
       .add("value", obj("city" -> fromString("Lublin"), "street" -> fromString("Lipowa"))))
+    val testRecordJson = parser.parse(s"""{"keySchemaId":null,"valueSchemaId":$id,"consumerRecord":${consumerRecord.noSpaces} }""").rightValue
```
Comment (mproch) [inline engine/flink/schemed-kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/schemedkafka/source/flink/TestFromFileSpec.scala:61]: let's use json API here

**Review by mproch — COMMENTED** [review]
Cool, I've left some comments

File: utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/RecordFormatter.scala [hunk 2]
```diff
@@ -1,54 +1,45 @@
...
-
-  protected def parseRecord(topic: String, bytes: Array[Byte]): ConsumerRecord[Array[Byte], Array[Byte]]
+  protected def formatRecord(record: ConsumerRecord[Array[Byte], Array[Byte]]): TestRecord
 
-  protected def testDataSplit: TestDataSplit
-
-  def prepareGeneratedTestData(records: List[ConsumerRecord[Array[Byte], Array[Byte]]]): Array[Byte] = {
-    testDataSplit.joinData(records.map(formatRecord))
-  }
+  def parseRecord(topic: String, testRecord: TestRecord): ConsumerRecord[Array[Byte], Array[Byte]]
```
[the remark this answers is not in the document]
Comment (jedrz) [reply utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/RecordFormatter.scala:20]: I think topic can still be found useful sometimes, for example one decide to save to file only value of an Avro record, then topic can be used to fetch latest schema to recreate the consumer record's value.

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerApp.scala [hunk 3]
```diff
@@ -141,12 +142,13 @@ trait NusskanckerDefaultAppRouter extends NusskanckerAppRouter {
 
     val newProcessPreparer = NewProcessPreparer(typeToConfig, additionalProperties)
 
-    val systemRequestTimeout = system.settings.config.getDuration("akka.http.server.request-timeout")
+    val systemRequestTimeout = system.settings.config.getDuration("akka.http.server.request-timeout").toMillis millis
```
[the remark this answers is not in the document]
Comment (jedrz) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerApp.scala]: Replaced with `Timeout`.

File: utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/RecordFormatter.scala (hunk 2 above)
[the remark this answers is not in the document]
Comment (mproch) [reply utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/RecordFormatter.scala:20]: Ok, let's leave it for now :+1:

**Review by mproch — APPROVED** [review]
LGTM :+1: Let's continue with the change!

**Pull request merged**